src/platform/packages/shared/kbn-esql-ast/src/__tests__/autocomplete.ts

```diff
@@ -75,9 +75,13 @@ export const suggest = (
 ): Promise<ISuggestionItem[]> => {
   const innerText = query.substring(0, offset ?? query.length);
   const correctedQuery = correctQuerySyntax(innerText);
-  const { ast } = parse(correctedQuery, { withFormatting: true });
+  const { ast, root } = parse(correctedQuery, { withFormatting: true });
```

Comment:
I started to avoid using parse since it reports it to me as derecated and use Parser.parse... if and when possible